.gitignore

```
@@ -41,3 +41,5 @@ release/
 backend-assets/
 prepare
 /ggml-metal.metal
+backend/rust/target/
+backend/rust/*.lock
```

```suggestion
target/
Cargo.lock
```
This should be simpler, also `git rm backend/rust/Cargo.lock`

---

Thanks.